src/DynamoCore/Models/DynamoModelCommands.cs

```diff
@@ -40,6 +40,7 @@ public void ExecuteCommand(RecordableCommand command)
         }
         
         private PortModel activeStartPort;
+        private PortModel[] activeStartPorts;
```

Comment:
I think it can be done. If we remove `activeStartPort` though, the original function that handles single connection without Ctrl press, which is `BeginConnection()` will have to be amended since it uses the variable.